lib/dotenv/log_subscriber.rb

```diff
@@ -24,7 +24,7 @@ def load(event)
     def update(event)
       diff = event.payload[:diff]
       changed = diff.env.keys.map { |key| color_var(key) }
-      debug "Set #{changed.to_sentence}" if diff.any?
+      debug "Set #{changed.join(", ")}" if diff.any?
```

Comment:
```suggestion
      debug { "Set #{changed.join(", ")}" } if diff.any?
```

This could be another improvement when the log level is to something less verbose, right?

Reply:
I think these are usually called only once (on boot) and does not matter.

Reply:
Alright, the PR made me think it's called multiple times. But apparently the slowdown came just from booting i18n. 😬

Reply:
> But apparently the slowdown came just from booting i18n.

Yes that's the reason why I proposed this change. Dotenv shouldn't have any impact post-boot 👍